src/components/layout/nav.js

```diff
@@ -494,7 +497,7 @@ const NavDesktop = ({ base, is_ppc_redirect }) => {
                     <NavLink onClick={handleTradeClick}>
                         <StyledButton
                             aria-label={localize('Trade')}
-                            active={is_trade_open}
+                            active={currentPage == 'trade' || is_trade_open}
```

Comment:
would it differ if we used `===`?

Reply:
@AbdallaMourad  behaviour-wise should be similar as object.keys return a string. Fixed in the next commits~